groclient/client.py

```diff
@@ -961,49 +961,10 @@ def get_descendant_regions(
     ):
         """Look up details of all regions of the given level contained by a region.
 
-        This method is deprecated, and should be replaced by :meth:`~.get_descendant` which has
-        been updated to include the `descendant_level` and `include_historical` parameters.
+        This functionality of this method can also be achieved by using :meth:`~.get_descendant` 
```

Comment:
Maybe more explicitly - This function has been moved to `get_descendant` .

Reply:
ok updated the language